subworkflows/local/allele_specific_calling.nf

```diff
@@ -89,10 +99,38 @@ workflow ALLELE_SPECIFIC_CALLING {
 
         BCFTOOLS_MERGE.out.merged_variants
             .mix( RENAME_FILES.out.output )
-            .set { ch_vcf }
+            .set { ch_vcf_merged }
+
+        TABIX_TABIX( ch_vcf_merged )
+
+        ch_in_split_multi=ch_vcf_merged.join(TABIX_TABIX.out.tbi)
+        SPLIT_MULTIALLELICS(ch_in_split_multi, ch_fasta)
+        TABIX_AFTER_SPLIT(SPLIT_MULTIALLELICS.out.vcf)
+
+        ch_remove_dup_in = SPLIT_MULTIALLELICS.out.vcf.join(TABIX_AFTER_SPLIT.out.tbi)
+        REMOVE_DUPLICATES(ch_remove_dup_in, ch_fasta)
+        TABIX_REMOVE_DUP(REMOVE_DUPLICATES.out.vcf)
+
+        ch_genome_chrsizes.flatten().map{chromsizes ->
+            return [[id:variant_caller], chromsizes]
+            }
+            .set { ch_varcallerinfo }
+
+        ADD_VARCALLER_TO_BED (ch_varcallerinfo).gz_tbi
+            .map{meta,bed,tbi -> return [bed, tbi]}
+            .set{ch_varcallerbed}
+
+        REMOVE_DUPLICATES.out.vcf
+            .join(TABIX_REMOVE_DUP.out.tbi)
+            .combine(ch_varcallerbed)
+            .combine(ch_foundin_header)
+            .set { ch_annotate_in }
+
+        BCFTOOLS_ANNOTATE(ch_annotate_in)
+
+        TABIX_ANNOTATE(BCFTOOLS_ANNOTATE.out.vcf)
```

Comment:
I would move this to the `call_variants` subworkflow.

Reply:
@jemten I see why, but the reason I put it here to make it more efficient because if its run after the vcf is merged, it would only have to be run once per case instead of once per sample.

Reply:
Done!